dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -779,6 +779,32 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
     assert_equal 0, @teacher.scripts.size
   end
 
+  test 'creating a section with a coteacher adds both teachers' do
```

Comment:
I'd like to see assertions that the original teacher is in active status and the coteacher is in invited status. Nice test coverage otherwise!